src/solver/type_getters.jl

```diff
@@ -352,6 +352,11 @@ function get_initial_condition(parsed_args)
             parsed_args["external_forcing_file"],
             parsed_args["cfsite_number"],
         )
+    elseif parsed_args["initial_condition"] == "ExternalTV"
```

Comment:
```suggestion
    elseif lowercase(parsed_args["initial_condition"]) == "externaltv"
```
will allow users to miscapitalize the option and the code will still work.